.github/workflows/platform-test.yaml

```diff
@@ -65,3 +65,44 @@ jobs:
       - run: mise run ci:gotest
       - run: mise run ci:catalog-test
       - run: mise run build:flow-schema
+  dekaf-test:
```

Comment:
Had to separate this out because `mise run ci:nextest-run` and/or `mise run ci:catalog-test` are leaving the control-plane in an unclean state, so I figured it would be simpler to fully isolate things since Dekaf's e2e tests also create/delete specs, publish to the data-plane, etc.